pkg/vault/test/foundry/DynamicFeePoolTest.t.sol

```diff
@@ -197,7 +214,8 @@ contract DynamicFeePoolTest is BaseVaultTest {
     }
 
     function testSwapChargesFees__Fuzz(uint256 dynamicSwapFeePercentage) public {
-        dynamicSwapFeePercentage = bound(dynamicSwapFeePercentage, 0, 1e18);
+        // Cannot be 100%, or it would do a zero amount swap, which is below the minimum.
+        dynamicSwapFeePercentage = bound(dynamicSwapFeePercentage, 0, 0.9999e18);
```

Comment:
I think we can put 0.9999e18 to a constant

Reply:
Well, we're trying to avoid having to do this (per @danielmkm's comment), by allowing 0 swaps (as we do for liquidity operations). This permits this to be 1 again.

However, it's arguable whether this is an improvement, as there is still a "banned" range near 100%. It's just that exactly 100% is now allowed. The UX issue with not allowing 100 is you'd have to explain that, and calculate the top value (which varies by pool state).

However, if you now increased the swap fee towards 100%, it would fail at some point (for a range < 100 but > 99.9x%), then work again at 100%. We'd also have to document this.

So the question is how is this used? Are they always just going to set 100% for this LVR case, whereas other use cases will set much lower values? Then it might be ok. But if we have a use case where it gradually increases the fee until it reaches 100%, this is arguably worse than not allowing 0.

(Also, is allowing 0 swaps a security hole making this entire thing pointless?)